templates/frag/fromJson.hbs

```diff
@@ -13,10 +13,12 @@ Output a reference to a FromJsonFunction function.
 {{prefix}}stringFromJson
 {{~else if (isBinary)~}}
 {{prefix}}binaryFromJson
+{{~else if (isFile)~}}
+{{prefix}}fileFromMultipart
```

Comment:
Thi is in the `fromJson` frag so we can't support the `FILE` type in JSON... so let's change this to `unsupportedFromJson`